client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -49,12 +48,20 @@ const ProfileInfoEdit = () => {
     setValidated(true);
   };
 
-  const profileDeleted = () => {
-    // TODO: Delete profile function is not implemented yet in ProfileInfoEdit.js
-    toast.error('Profile has not been deleted');
-  };
+  const [deleteAlert, setdeleteAlert] = useState(false);
+
+  // const updateProfile = (teamId, profileData) => {
+  //   try {
+  //     dispatch(updateTeam(teamId, profileData))
+  //     toast.success('Profile has been successfully updated');
+  //   } catch (error) {
+  //     console.error(error);
+  //     toast.error('Profile has not been updated');
+  //   }
+  // };
```

Comment:
Maybe comments can be taken out?

Reply:
I got this from a previous pull but yeah will surely take this out!